account_avatax/models/account_tax.py

```diff
@@ -17,6 +17,7 @@ def _get_avalara_tax_domain(self, tax_rate, doc_type):
         return [
             ("amount", "=", tax_rate),
             ("is_avatax", "=", True),
+            ("company_id", "=", self.env.user.company_id and self.env.user.company_id.id),
```

Comment:
This is safe, AFAICT
```suggestion
            ("company_id", "=", self.env.user.company_id.id),
```